Dockerfile-ubuntu

```diff
@@ -22,6 +22,9 @@ RUN curl -L https://go.dev/dl/go${GO_VERSION}.linux-amd64.tar.gz | tar -C $LOCAL
 # Install ignite
 RUN curl -L https://get.ignite.com/cli@v${IGNITE_VERSION}! | bash
 
+# Clean apt cache
+RUN apt-get cache
```

Comment:
Good idea. 
Is it not `apt-get clean`?

Reply:
Hi @xavierlepretre, you are right :)

I should have updated the file